db/migrate/20230429120200_remove_automatically_approve_collections.rb

```diff
@@ -0,0 +1,43 @@
+class RemoveAutomaticallyApproveCollections < ActiveRecord::Migration[6.0]
+  def up
+    if Rails.env.staging? || Rails.env.production?
+      database = Preference.connection.current_database
+
+      puts <<~PTSOC
+        Schema Change Command:
```

Comment:
Just one minor thing: can you add spacing between the descriptions and the actual commands? For example:

https://github.com/otwcode/otwarchive/blob/56ef6bb7390df891814cc1843db627cc80419c39/db/migrate/20220918015954_add_allow_collection_invitation_to_preference.rb#L7-L19